src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/FixedAddressValueTypeAttribute.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [Serializable]
+    [AttributeUsage(AttributeTargets.Field)]
+    public sealed class FixedAddressValueTypeAttribute : Attribute
+    {
+        public FixedAddressValueTypeAttribute() { }
```

Comment:
Does this now actually require compiler work on the CoreRT side to respect this attribute? Do we need a tracking issue?

In other words: is there a reasonable expectation that various Managed C++ attributes that got added to netstandard without regard of their usefulness or complexity involved will work? Cc @danmosemsft @fadimounir